command/acl/authmethod/update/authmethod_update.go

```diff
@@ -64,6 +66,23 @@ func (c *cmd) init() {
 		"",
 		"A description of the auth method.",
 	)
+	c.flags.StringVar(
```

Comment:
You duplicated this in the move and I think the first one is the one you didn't want.